docs/pages/demo/timepicker/KeyboardTimePicker.example.jsx

```diff
@@ -1,11 +1,11 @@
 import React, { useState } from 'react';
-import { KeyboardTimePicker } from '@material-ui/pickers';
+import { TimePicker } from '@material-ui/pickers';
```

Comment:
I'm wondering, should we document this approach instead? 

```suggestion
import TimePicker from '@material-ui/pickers/TimePicker';
```

At the same time, I was wondering about changing the demos of http://material-ui.com/, to use the Pull Request's approach. I'm curious to hear your thoughts on this.

Reply:
We have different build system. esm build allowed only by importing from the root. P.S tree shaking works

Reply:
OK, good to know.